### DISCUSSION TITLE
Proposal for Releasing Zul'Aman Prior to Black Temple Based on Logical Progression Patterns

### DISCUSSION THREAD

**Discussion by lddc1993** [chromiecraft/chromiecraft#6171] [Suggestions]
To ensure a smooth and meaningful progression for players in "The Burning Crusade", we should look to the raid release structures of later expansions as a model. Specifically, "Wrath of the Lich King" (WoTLK) presents a compelling argument for the order of raid releases.

In WoTLK, raids were structured to lead players seamlessly from one tier to the next. Naxxramas, as a re-introduced raid, led players into Ulduar, and subsequently, the Argent Tournament and then Icecrown Citadel. Each step of the progression had a distinct gear level, ensuring that each raid tier had value and relevance.

Given the positioning of ZA gear between T5 and T6:
 - Releasing ZA after SSC and TK (T5 raids) but before BT will give guilds a smoother progression curve.
 - It'll serve as a bridge between the challenges of T5 and the more demanding T6 in BT.
 
While it could be argued that ZA might fit elsewhere in the raiding timeline, it's crucial to look at its relative difficulty and the itemization of its gear.
Placing ZA after BT would create an unnecessary and confusing jump back in terms of gear progression.
Using the WoTLK model, it's similar to releasing Naxxramas after Ulduar, which would have been counter-intuitive.
 
ZA's role isn't merely about its loot but also its place in the broader story and raiding landscape of TBC.
Much like how Naxxramas in WoTLK offered a transition into the more challenging content of Ulduar, ZA can serve a similar role, transitioning players from the challenges of T5 into the more demanding world of T6.

Optional Proposal for a Balanced Player Experience:
   - Initial Release: Launch ZA with a 7-day lockout after the completion of T5 raids (SSC and TK), offering a logical step in progression before diving into T6 with BT.
   - Post-BT Release: Revert the ZA lockout to 3 days after the release of BT, allowing it to serve as a catch-up mechanism for players who joined late or guilds that need to bolster their gear before diving deeper into T6 content.
 
Basing our raid release structure on the progression patterns evident in later expansions like WoTLK ensures a meaningful and logical player experience. Positioning Zul'Aman between T5 and T6 raids offers a natural and continuous progression that respects both the challenges of each raid and the players' investment in their characters.

**Comment by Amrx2** [comment]
Hi there, thank you for your suggestion. Whilst looking strictly at item levels your wording around gear is correct, in reality stats and itemization differs from that and certain items in ZA are BiS compared to what you can get raids such as BT. Our progression reflects that of the original release of the game and ZAs position of a catchup raid more, given that we will not be making this change but thank you for your suggestion. 

Cheers,
Construct
